# Mapping Configuration
---
# Survey Data ##################################################################
# # Where the blender track mesh is zeroed
UTMdatum: # 2 layer nested dictionary
    E: 313617.45721616584 
    N: 4314077.05485446
#
# # Locations of stripe lat/lon survey points
survey_center_locs: 
- 'survey/stripes/TurnerFairbanks_RoadEdge.txt'
#
# # Locations of lane center lat/lon survey points
survey_stripe_locs:
- 'survey/centers/TurnerFairbanks_Centerline.txt'
#
# # File location for mesh of track
# # the syntax denotes a location in the ROS package
track_mesh_resource: None
#
# # File location for mesh of lane markings
# # syntax denotes a location in the ROS package; None to turn off
marking_mesh_resource: None
# marking_mesh_resource: None
##############################################################################